pkg/translate/service.go

```diff
@@ -68,35 +70,96 @@ func (t *TranslateServiceServer) ListServices(
 	ctx context.Context,
 	req *translatev1.ListServicesRequest,
 ) (*translatev1.ListServicesResponse, error) {
-	return &translatev1.ListServicesResponse{}, nil
+	services, err := t.repo.LoadServices(ctx)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, err.Error())
+	}
+
+	return &translatev1.ListServicesResponse{Services: servicesToProto(services)}, nil
+}
+
+// ---------------------CreateService-------------------------------
+
+type createServiceParams struct {
+	service *model.Service
+}
+
+func (c *createServiceRequest) parseParams() (*createServiceParams, error) {
+	if c == nil {
+		return nil, errors.New("request is nil")
+	}
+
+	if c.Service == nil {
+		return nil, errors.New("service is nil")
+	}
+
+	service, err := serviceFromProto(c.Service)
+	if err != nil {
+		return nil, fmt.Errorf("parse service: %w", err)
+	}
+
+	return &createServiceParams{service: service}, nil
+}
+
+func (t *TranslateServiceServer) CreateService(
+	ctx context.Context,
+	req *translatev1.CreateServiceRequest,
+) (*translatev1.Service, error) {
+	createReq := (*createServiceRequest)(req)
+
+	params, err := createReq.parseParams()
```

Comment:
Let's simplify this abstraction - it does not add anything. Consider using functional programming paradigms first.

Please apply it to all request parsing.

```suggestion
	params, err := parseCreateServiceRequest(req)
```